recipes/avahi/all/conanfile.py

```diff
@@ -26,75 +30,68 @@ class AvahiConan(ConanFile):
         "fPIC": True
     }
 
-    _autotools = None
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def layout(self):
+        basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("glib/2.68.3")
-        self.requires("expat/2.4.1")
+        self.requires("glib/2.74.0")
+        self.requires("expat/2.4.9")
         self.requires("libdaemon/0.14")
-        self.requires("dbus/1.12.20")
+        self.requires("dbus/1.15.0")
         self.requires("gdbm/1.19")
         self.requires("libevent/2.1.12")
 
     def validate(self):
-        if self.settings.os != "Linux" or tools.cross_building(self):
+        if self.info.settings.os != "Linux":
             raise ConanInvalidConfiguration("Only Linux is supported for this package.")
 
     def configure(self):
-        del self.settings.compiler.cppstd
-        del self.settings.compiler.libcxx
+        try:
+            del self.settings.compiler.cppstd
```

Comment:
This is way better then doing them one at a time....

Reply:
ikr? :)